app/components/services/servicesController.js

```diff
@@ -72,15 +72,28 @@ function ($scope, $stateParams, $state, Service, ServiceHelper, Messages, Pagina
 
   function fetchServices() {
     $('#loadServicesSpinner').show();
-    Service.query({}, function (d) {
-      $scope.services = d.map(function (service) {
-        return new ServiceViewModel(service);
+    $q.all({
+      services: Service.query({}).$promise,
+      tasks: Task.query({filters: {'desired-state': ['running']}}).$promise,
+      nodes: Node.query({filters: {membership: ['accepted']}}).$promise,
```

Comment:
So we need to do filter client side on portainer, not on the query (or remove filter waiting for better solution !)